op-service/txmgr/txmgr.go

```diff
@@ -140,6 +140,7 @@ type SimpleTxManager struct {
 	backend ETHBackend
 	l       log.Logger
 	metr    metrics.TxMetricer
+	gpe     GasPriceEstimator
```

Comment:
can we call it `GasEstimationFunc` or something more verbose? `cfg` is a common shortening, but `gpe` is not familiar to me.